teammapper-frontend/src/app/modules/application/components/dialog-about/dialog-about.component.ts

```diff
@@ -35,7 +35,18 @@ export class DialogAboutComponent {
     if (confirm(this.translateService.instant('MODALS.INFO.CONFIRM_DELETE'))) {
       await this.mapSyncService.deleteMap(await this.mapAdminId);
       await this.storageService.remove(this.map.uuid);
-      window.location.reload();
+
+      const url = new URL(window.location.href);
```

Comment:
can't we use the router.state here as well? I think its a more unified way of accessing the url

Reply:
Yeah that was my preferred way of doing it since `router.navigate` is a much cleaner way of doing things and allows us to separate URL, query params and fragment, but wasn't sure if it was worth it for a one off (esp. considering the component doesn't have the router yet)

Reply:
Added in this.router.navigate()